src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -91,56 +91,56 @@ be specified.
-| SoftwareFilters    | REQUIRED. [Object][] of temporal software filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs. E.g., `{"Anti-aliasing filter": {"half-amplitude cutoff (Hz)": 500, "Roll-off": "6dB/Octave"}}`.                          |
+| **Key name**       | **Requirement level** | **Data type**                        | **Description**                                                                                                                                                                                                                                                                                                                                        |
+| ------------------ | --------------------- | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| EEGReference       | REQUIRED              | [string][]                           | General description of the reference scheme used and (when applicable) of location of the reference electrode in the raw recordings (e.g., "left mastoid", "Cz", "CMS"). If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the channels.tsv file. |
+| SamplingFrequency  | REQUIRED              | [number][]                           | Sampling frequency (in Hz) of all the data in the recording, regardless of their type (e.g., 2400).                                                                                                                                                                                                                                                    |
+| PowerLineFrequency | REQUIRED              | [integer][] or `"n/a"`               | Frequency (in Hz) of the power grid at the geographical location of the EEG instrument (i.e., 50 or 60).                                                                                                                                                                                                                                               |
+| SoftwareFilters    | REQUIRED              | [object][] of [objects][] or `"n/a"` | [Object][] of temporal software filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs. E.g., `{"Anti-aliasing filter": {"half-amplitude cutoff (Hz)": 500, "Roll-off": "6dB/Octave"}}`.                        |
```

Comment:
Adding backticks to `channels.tsv` and two spaces/dashes to all other lines.

```suggestion
| **Key name**       | **Requirement level** | **Data type**                        | **Description**                                                                                                                                                                                                                                                                                                                                          |
| ------------------ | --------------------- | ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| EEGReference       | REQUIRED              | [string][]                           | General description of the reference scheme used and (when applicable) of location of the reference electrode in the raw recordings (e.g., "left mastoid", "Cz", "CMS"). If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the `channels.tsv` file. |
| SamplingFrequency  | REQUIRED              | [number][]                           | Sampling frequency (in Hz) of all the data in the recording, regardless of their type (e.g., 2400).                                                                                                                                                                                                                                                      |
| PowerLineFrequency | REQUIRED              | [integer][] or `"n/a"`               | Frequency (in Hz) of the power grid at the geographical location of the EEG instrument (i.e., 50 or 60).                                                                                                                                                                                                                                                 |
| SoftwareFilters    | REQUIRED              | [object][] of [objects][] or `"n/a"` | [Object][] of temporal software filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs. E.g., `{"Anti-aliasing filter": {"half-amplitude cutoff (Hz)": 500, "Roll-off": "6dB/Octave"}}`.                          |
```

Reply:
Would you consider having PowerLineFrequency a number rather than an integer. Line noise isn't always exactly at its nominal value. 

Is there any planning for having more specific validation criteria such as "positive", "non-negative", "<X", etc.  Just curious?

Reply:
> Adding backticks to channels.tsv and two spaces/dashes to all other lines.

thanks

> Would you consider having PowerLineFrequency a number rather than an integer. Line noise isn't always exactly at its nominal value.

True, will adjust.

> Is there any planning for having more specific validation criteria such as "positive", "non-negative", "<X", etc. Just curious?

that'd be pretty cool actually. We are using the JSON schema in the validator, I haven't looked into whether or not such validation is possible

Reply:
indeed, validating that "PowerLineFrequency" is positive we can use `"exclusiveMinimum": 0` in the JSON schema, see the [docs for that](https://json-schema.org/understanding-json-schema/reference/numeric.html#range). I am making a PR to bids-validator to do just that.